services/web/README.md

```diff
@@ -0,0 +1,8 @@
+# Web service
+
+The web service embeds and serves the static files of the [Infinite Scale Web Client](https://github.com/owncloud/web).  
+It also provides a minimal API for branding functionality like changing the logo.
+
+# Custom compiled Web assets
+
+If you want to use your custom compiled web assets then you can do that by setting the `WEB_ASSET_PATH` variable to point to your compiled files.
```

Comment:
```suggestion
If you want to use your custom compiled web client assets instead of the embedded ones, then you can do that by setting the `WEB_ASSET_PATH` variable to point to your compiled files.
```

Reply:
Is there any more info how one can get to own compiled ones?

Reply:
Based on the discussion with @C0rby, I added links for more details.